performance_manager/lib/l2_dwell_travel_times.py

```diff
@@ -0,0 +1,165 @@
+import sqlalchemy as sa
+
+from .logging_utils import ProcessLogger
+from .postgres_utils import DatabaseManager
+from .postgres_schema import (
+    VehiclePositionEvents,
+    FullTripEvents,
+    DwellTimes,
+    TravelTimes,
+)
+
+
+def dwell_times(db_manager: DatabaseManager) -> None:
+    """
+    perform update on dwell times table
+
+    executed as delete and insert
+    """
+    process_logger = ProcessLogger("update l2 dwell times")
```

Comment:
i've been using snake case on the process names to make sure splunk can index them correctly.

Reply:
incorporated